scripts/runShiftLeftScan.sh

```diff
@@ -1,3 +1,14 @@
-# Generates results files in the folder reports/. The ShiftLeftScanReader processes the file: reports/scan-full-report.json
-docker run --rm -e "WORKSPACE=${PWD}" -v ~/.m2:/.m2 -v ${PWD}:/app shiftleft/scan scan --src /app --type java
+#!/usr/bin/env sh
 
+# Check for install/updates at https://github.com/ShiftLeftSecurity/sast-scan
+
+benchmark_version=$(scripts/getBenchmarkVersion.sh)
+shiflteft_version="2.0.3" # it's not (yet) possible to get the release version so we just assume it
+result_file="results/Benchmark_$benchmark_version-shiftleftscan-v$shiflteft_version.json"
+
+mkdir -p .shiftlefscan-reports
+
+docker run --rm -e "WORKSPACE=${PWD}" -v ~/.m2:/.m2 -v "$PWD":/app -v "$PWD/.shiftlefscan-reports":/app/reports shiftleft/scan scan --src /app --type java
+mv .shiftlefscan-reports/scan-full-report.json "$result_file"
+
+rm -rf .shiftlefscan-reports
```

Comment:
If this requires Docker only, can you verify Docker is installed first and if not, say it's required, and then exit.

Reply:
Docker is required now and typo is fixed. Can't reproduce the error though. I guess this came because of the "broken" benchmark script? Can you try it again?